test/NPacMan.Game.Tests/GameTests/GhostBuilder.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using NPacMan.Game.GhostStrategies;
+using NPacMan.Game.Tests.GhostStrategiesForTests;
+
+namespace NPacMan.Game.Tests.GameTests
+{
+    public class GhostBuilder
+    {
+        private int _numberOfCoinsRequiredToExitHouse = 0;
+        private IGhostStrategy _chaseStragtegy = new StandingStillGhostStrategy();
+        private CellLocation _scatterTarget = CellLocation.TopLeft;
+        private Direction _direction = Direction.Left;
+        private CellLocation _location = new CellLocation(0, 0);
+        private static readonly Queue<string> _names = new Queue<string>(
+            Enumerable.Range(0, 1000).Select(i => $"Ghost{i}"));
```

Comment:
Is this better than just keeping track of the number of ghosts created?

Reply:
This could just be a random generate GUID now, as I changed all the magic strings to `ghost.name`